src/tarski/reachability/clingo_wrapper.py

```diff
@@ -8,8 +8,31 @@
 
 from ..errors import CommandNotFoundError, ExternalCommandError, OutOfMemoryError, OutOfTimeError, ArgumentError
 from ..utils import command as cmd
+from importlib.util import find_spec
+
+
+def get_gringo_command():
+    """Assemble the command to run gringo
+
+    :return: A list depicting the command with arguments
+    :rtype: list of String / None(if command could not be assembled)
+    """
+    command = None
+    if find_spec("clingo"):
+        command = [sys.executable, os.path.join(Path(__file__).parent.absolute(), "gringo.py")]
```

Comment:
Are we spawning a new process here on purpose? Do we need that, or could we simply `import` our clingo wrapper and invoke `clingo_main`?

Reply:
I tried it but I couldn't redirect `stdout` and `stderr` properly. Technically, we just need to use [`redirect_stdout`](https://docs.python.org/3/library/contextlib.html#contextlib.redirect_stdout) but it wouldn't work.